engine/class_modules/sc_death_knight.cpp

```diff
@@ -6694,7 +6695,7 @@ struct swarming_mist_damage_t : public death_knight_spell_t
     if ( swarming_mist_energize_tick < swarming_mist_energize_target_cap )
     {
       p() -> resource_gain( RESOURCE_RUNIC_POWER,
-                 p() -> covenant.swarming_mist_energize->effectN( 1 ).resource( RESOURCE_RUNIC_POWER ),
+                 swarming_mist_energize->effectN( 1 ).resource( RESOURCE_RUNIC_POWER ),
```

Comment:
I meant that you could just store the value from .resource as an int, not even the whole spelldata object

Reply:
that would.... make far too much sense.  I thought it odd from an efficiency standpoint to move the spelldata, but figured you wouldn't suggest it without reason.  Now it makes a lot more sense.